pkgs/data/fonts/inter/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchzip }:
 
 let
-  version = "3.5";
+  version = "3.7";
 in fetchzip {
```

Comment:
Wouldn't be it better to use fetchFromGitHub?

Reply:
Answered in other comment, sorry about that.  In short: can't use fetchFromGitHub to grab release assets unfortunately, but if we could yes that's preferred :).